docs/docs/develop/setup.md

```diff
@@ -54,10 +60,17 @@ tags:
 | `--version`       | Show version number                        | boolean                                                         |
```

Comment:
I think we can remove `--version` from this list, appears to only output the npm version

Reply:
you have to run these commands using `npm run new -- --version` that will give you the version of `pfe-tools` I believe. I believe you can think of this as the generator version.  I actually at one point had removed it myself, then added the Example: below

Reply:
I take that back version doesn't work. Hmm could have sworn I ran that as the example /shrug.  Nevermind will remove it.